requirements.txt

```diff
@@ -10,6 +10,7 @@ chardet==3.0.4            # via requests
 click==7.0                # via flask
 flask-basicauth==0.2.0
 flask==1.1.1
+Flask-Limiter==1.1.0
```

Comment:
I'm going to update this to set the package in requirements.in and compile with transitive dependencies to this requirements.txt file.